mne/utils/docs.py

```diff
@@ -1963,18 +1963,18 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 """
 
 _method_psd = """\
-method : 'welch' | 'multitaper'{}
+method : ``'welch'`` | ``'multitaper'``{}
     Spectral estimation method. ``'welch'`` uses Welch's method
     :footcite:`Welch1967`, ``'multitaper'`` uses DPSS tapers
     :footcite:`Slepian1978`.{}
 """
 docdict['method_plot_psd_auto'] = _method_psd.format(
-    " | 'auto'",
+    " | ``'auto'``",
     (" ``'auto'`` (default) uses Welch's method for continuous data and "
      "multitaper for :class:`~mne.Epochs` or :class:`~mne.Evoked` data.")
 )
 docdict['method_psd'] = _method_psd.format('', '')
-docdict['method_psd_auto'] = _method_psd.format(" | 'auto'", '')
+docdict['method_psd_auto'] = _method_psd.format(" | ``'auto'``", '')
```

Comment:
are these changes consistent with the rest of the docstrings?

Reply:
They are, if I did not make a mistake. I'll check and share the links to the render doc as soon as it's built.

Reply:
in fact we're not systematic about using double-backticks in docstrings when specifying specific strings allowed as parameter values.  In a few cases they are mandatory (I think if it's a tuple maybe?).  But in most cases we don't include the double backticks (see, for example, `outlines : 'head' | 'skirt' | dict | None` or `ch_type : ‘mag’ | ‘grad’ | ‘planar1’ | ‘planar2’ | ‘eeg’ | None`

However, it does look better the way you've done it here, so maybe we should slowly move toward including double-backticks.